src/documents/components/ReviewDetail/ReviewVersions.js

```diff
@@ -0,0 +1,213 @@
+import {
+  Button,
+  Header,
+  Icon,
+  Label,
+  Popup,
+  Segment,
+  Table,
+} from 'semantic-ui-react';
+import {Link, withRouter} from 'react-router-dom';
+
+import FileActionButtons from '../FileActionButtons/FileActionButtons';
+import KfId from '../../../components/StudyList/KfId';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {fileSortedVersions} from '../../utilities';
+import {fileTypeDetail} from '../../../common/enums';
+
+/**
+ * Displays data review attached document version list with action buttons
+ */
+const ReviewVersions = ({
+  studyId,
+  reviewNode,
+  updateReview,
+  updateError,
+  setAddFileOpen,
+  downloadFileMutation,
+  canAddFile,
+}) => {
+  const groupType = ['PDA', 'GWO', 'SEQ', 'S3S'];
+  const groupVersions = () => {
+    const versions = reviewNode ? reviewNode.versions.edges : [];
+    var versionGroups = {
+      PDA: [],
+      GWO: [],
+      SEQ: [],
+      S3S: [],
+      OTHER: [],
+    };
+    if (versions.length > 0) {
+      versions.forEach(({node}) => {
+        if (groupType.includes(node.rootFile.fileType)) {
+          versionGroups[node.rootFile.fileType].push(node);
+        } else {
+          versionGroups.OTHER.push(node);
+        }
+      });
+    }
+    return versionGroups;
+  };
+  const groupedVersions = groupVersions();
+
+  return (
+    <Segment basic className="noMargin">
+      <Header as="h4" color="grey">
+        Documents in Review{' '}
+        {updateReview && canAddFile && (
+          <Popup
+            content="Add documents to current data review"
+            position="right center"
+            inverted
+            trigger={
+              <Button
+                size="mini"
+                labelPosition="left"
+                className="ml-15 text-primary"
+                onClick={() => {
+                  setAddFileOpen(true);
+                }}
+              >
+                <Icon name="add" />
+                Add Documents to Review
+              </Button>
+            }
+          />
+        )}
+      </Header>
+      {groupType.map(type => (
+        <Table key={type}>
+          <Table.Header>
+            <Table.Row>
+              <Table.HeaderCell colSpan="4">
```

Comment:
File types also have a url that points to documentation on that type. It may be good to include a link to that here.